src/components/file-uploader/file-uploader.js

```diff
@@ -7,6 +7,29 @@ import handles from '../../globals/js/mixins/handles';
 import eventMatches from '../../globals/js/misc/event-matches';
 import on from '../../globals/js/misc/on';
 
+/**
+ * We're using a pattern where we style an input's label to look like a button.
+ * Consequently we need to add proper keyboard accessibility to the label in
+ * order to mimic if it were an input button -- also worth noting is we hide the
+ * actual input from tab order in the markup.
+ */
+if (document.readyState === 'loading') {
+  document.addEventListener('DOMContentLoaded', () => {
+    const uploadLabel = document.getElementById('input-button');
+    const uploadInput = document.getElementById('your-file-importer-id-here');
```

Comment:
Seems that the code here needs to be implemented in `FileUploader` vanilla class - Sent a private note.